pkg/controller/installation/core_controller.go

```diff
@@ -1000,25 +999,18 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 			return reconcile.Result{}, err
 		}
 
-		// Successful reconciliation of non-NetworkPolicy resources in the core controller ensures that NetworkPolicy
-		// is reconcilable (by enabling the availability of the API server and the creation of containing Tier). Therefore,
-		// to prevent a chicken-and-egg scenario, we only reconcile NetworkPolicy resources once we can confirm that all
-		// requirements to reconcile NetworkPolicy have been met.
-		//
-		// utils.IsV3NetworkPolicyReconcilable does not verify API server availability, so we take extra precaution
-		// when rendering components below.
-		if utils.IsV3NetworkPolicyReconcilable(ctx, r.client, networkpolicy.TigeraComponentTierName) {
-			includeV3NetworkPolicy = true
-
-			if !r.tierWatchReady.IsReady() {
-				r.status.SetDegraded("Waiting for Tier watch to be established", "")
-				return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
-			}
-
-			if !r.policyWatchesReady.IsReady() {
-				r.status.SetDegraded("Waiting for NetworkPolicy watches to be established", "")
-				return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
+		// Ensure the allow-tigera tier exists, before rendering any network policies within it.
+		if err := r.client.Get(ctx, client.ObjectKey{Name: networkpolicy.TigeraComponentTierName}, &v3.Tier{}); err != nil {
```

Comment:
Do we need / want a tier watch guarding this Get() call?

Reply:
Basically just conditionally execute this whole block based on whether the tier watch is established? 


```go
includeNetworkPolicy = false
if tierWatch.Ready() {
  if err := r.client.Get(tier); err != nil {
    // Handle error
  } else {
    // Tier exists.
    includePolicy = true
  }
}